ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/recovery/transactions/TransactionCache.java

```diff
@@ -311,6 +316,10 @@ private static void remove (Uid theUid)
         }
 	}
     }
+
+    public static enum ReplayPhaseReturnStatus {
+        NONE, ASSUMED_COMPLETED, CACHE_REMOVED
```

Comment:
I am not spotting the usage of the three different types here. Just whether it is still NONE or not